stdlib/int32.gr

```diff
@@ -138,11 +136,16 @@ export let rec mod = (x: Int32, y: Int32) => {
     let yabs = abs(yval)
     let mval = WasmI32.remS(xabs, yabs)
     let mres = WasmI32.sub(yabs, mval)
-    newInt32(if (WasmI32.ne(mval, 0n)) (if (WasmI32.ltS(yval, 0n)) WasmI32.sub(0n, mres) else mres) else 0n)
+    newInt32(
+      if (WasmI32.ne(mval, 0n)) if (WasmI32.ltS(yval, 0n)) WasmI32.sub(
+        0n,
+        mres,
+      ) else mres else 0n,
```

Comment:
this is crazy

Reply:
I had to do it like that to keep the else statements in the right place for the menhir parser I think, @ospencer ?

Reply:
This is now improved I think

Reply:
Fixed.